src/slider/__tests__/slider.test.js

```diff
@@ -1,248 +0,0 @@
-/*
-Copyright (c) 2018 Uber Technologies, Inc.
-
-This source code is licensed under the MIT license found in the
-LICENSE file in the root directory of this source tree.
-*/
-// @flow
-import React from 'react';
-import {mount} from 'enzyme';
-import {Slider, StyledThumb} from '../index.js';
```

Comment:
Is the thought behind deleting the `slider.test.js` file that the stateful component tests should cover the functionality?

Reply:
The functionality of slider is already tested in `react-range`.  So it doesn't make sense to test it twice.

`baseui/slider` just adds visual components on a top of it.